core/modules/czar/bin/startCzar.py

```diff
@@ -113,15 +103,19 @@ def main():
     log.debug("Configuration:\n%s", config.toString())
 
     if options.test == True:
+        # not working
         runParserTest()
         return
     elif options.testName:
+        # not working
         runNamedTest(options.testName)
         return
     elif options.sanityClient:
+        # not working
         client.runSanityClient()
         return
     elif options.makeIndex:
+        # not working
```

Comment:
I'm confused about this file, were these never working or is it because of your changes now?